### PR TITLE
Hit Skiddle Endpoint, closes #14

### LINKED ISSUE #14: Get live festival data from Skiddle API

### PR DESCRIPTION
**Pull request by louisditzel**
- Skiddle API /events is now hit when the client makes a request to the server /events.
- Framework now there to serve different event lists depending on context

### CODE REVIEW

File: config/api_keys.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"skiddle": {
 		"url": "https://www.skiddle.com/api/v1/",
-		"key": "/?api_key=4746dc555db14c2c5b8f52295ef28c08"
+		"key": "4746dc555db14c2c5b8f52295ef28c08"
```
Comment (JamesAdamHughes) [inline config/api_keys.json]: uhh i think we should remove this from git, maybe add config/\* to gitignore

File: controllers/events.js [hunk 2]
```diff
@@ -17,12 +18,40 @@ router.get('/event', function(req, res){
...
+            console.log("GET " + options.url);
+
+            //Send the request
+            request(options, function (error, reqResponse, body) {
+              if (error) throw new Error(error);
+              var contents = JSON.parse(body);
+              if (contents.error !== 0) {
+                console.log('ERROR ' + contents.errorcode + ': ' + contents.errormessage);
+              } else {
+                  response = contents.results;
```
Comment (JamesAdamHughes) [inline controllers/events.js:39]: can we cache the results, so when we refresh the page we don't have to hit skiddle everytime? only get the data the first time the /events endpoint is hit?

Comment (louisditzel) [inline controllers/events.js:39]: Yeah, we can do, and I think it's a good idea, but not sure it should necessarily be part of this pull req. This was more to get the end point integrated than to do anything fancy with it.

Comment (JamesAdamHughes) [inline controllers/events.js:39]: yep, fair point, i'll make an issue for it now

File: config/api_keys.json (hunk 1 above)
Comment (louisditzel) [inline config/api_keys.json]: Done

**Comment by JamesAdamHughes** [comment]
noice

**Pull request merged**